File: Hosts/TechChallenge.Spa/src/app/dashboard/dto/race-stat.js
var RaceStat = /** @class */ (function () {
    function RaceStat(id, name, status, raceTotalAmount, horseStats, start) {
        if (id === void 0) { id = 0; }
        if (name === void 0) { name = ''; }
        if (status === void 0) { status = ''; }
        if (raceTotalAmount === void 0) { raceTotalAmount = 0; }
        if (horseStats === void 0) { horseStats = []; }
        this.id = id;
        this.name = name;
        this.status = status;
        this.raceTotalAmount = raceTotalAmount;
        this.horseStats = horseStats;
        this.start = start;
    }
    return RaceStat;
}());
export { RaceStat };

File: Business/TechChallenge.Business.Common/Dto/RaceStat.cs
using System;
using System.Collections.Generic;
using Eml.Contracts.Entities;

namespace TechChallenge.Business.Common.Dto
{
    public class RaceStat : IEntityBase<int>
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Status { get; set; }

        public DateTime Start { get; set; }

        public double RaceTotalAmount { get; set; }

        public List<HorseStat> HorseStats { get; set; } = new List<HorseStat>();
    }
}
